crates/templates/src/manager.rs

```diff
@@ -181,6 +191,15 @@ impl TemplateManager {
         };
         let id = template.id();
 
+        if !install_template_list.is_empty() && !install_template_list.contains(&id.to_owned()) {
```

Comment:
The problem with a pure filtering approach is error handling.  If I write `install --id http-rsut` it will cheerily report "No templates installed" and the user is going "but where is my Rust template."  The user is unlikely to think of their command as filtering: they think they have _asked for_ a specific template.  So if we can't find the template they asked for, that feels like we should be telling them "hey we didn't find http-rsut" and maybe even "but may we interest you in this charming http-rust, very popular, very in right now."

Reply:
I agree that the preferred approach should be to let them know if we did not find a template that they asked for.